README.md

```diff
@@ -33,8 +33,12 @@ You can however set it to `rmw_fastrtps_dynamic_cpp` using the environment varia
 * Publication mode: `ASYNCHRONOUS_PUBLISH_MODE`
 
 However, it is possible to fully configure Fast DDS (including the history memory policy and the publication mode) using an XML file as described in [Fast DDS documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html).
```

Comment:
Hi, thanks for the PR, but the README still looks misleading to me. Line 35 reads like _"it is possible to **fully** configure Fast DDS using the XML file"_, which is not the case.

If only _History memory policy_ and _Publication mode_ can be configured with the XML, then I would expect a big warning right here, otherwise issues like #465 will keep piling up.

Reply:
Hi @jcaraban,

I thinks there's been a misuderstanding. It is indeed possible to fully configure Fast DDS using the XML, not just the *History memory policy* and *Publication mode*. What the README intends to convey is that to modify those 2 particular QoS, you will need both have a `DEFAULT_FASTRTPS_PROFILES.xml` file and to set `RMW_FASTRTPS_USE_QOS_FROM_XML=1`. Else, *History memory policy* and *Publication mode* will be set by RMW regardless of what your XML states